_locales/en/messages.json

```diff
@@ -158,8 +158,14 @@
     "description":"The button text which says 'Cancel'."
   },
   "saveFilePrompt": {
-    "message": "Do you want to save the file before closing?",
-    "description":"The text which prompts user if they want to save file before closing it."
+    "message": "$filename$ has been modified. Do you want to save it before closing?",
+    "description":"The text which prompts user if they want to save file before closing it.",
+    "placeholders": {
+      "filename": {
+        "content": "$1",
+        "example": "index.html"
```

Comment:
Ack

Reply:
Actually if I change it to a meaningfully named .txt file (like myfile.txt) it would require another round of translation. I figured html files would be a common use case for the app (and the name of the index.html name wouldn't change in different languages). Maybe I'll just use something like 'file.txt' and hope that Google translate is good enough since this is example text only that the user doesn't see.